docker-compose.yml

```diff
@@ -20,4 +20,6 @@ services:
       - BACKBLAZE_ACCESS_KEY
       - BACKBLAZE_SECRET
       - WASABI_ACCESS_KEY
-      - WASABI_SECRET
\ No newline at end of file
+      - WASABI_SECRET
+      - CEPH_ACCESS_KEY
+      - CEPH_SECRET
```

Comment:
Why are these prefixed with "CEPH"?

Reply:
Oh, yes. CEPH is generally what I have been using with ibm cloud storage for a past year. However, I see the problem here. Will make the PR with IBM_CLOUD_OBJECT prefix, inorder to resolve the issue.